### ISSUE TITLE
circular import

### ISSUE THREAD

**Issue by Rithsek99** [marieai/marie-ai#101]
**Describe the bug**
 when oc-gradio we get the follow error, potentially circular import. 

File "/home/rngem/Desktop/rms/marie-ai/workspaces/ocr-gradio/app.py", line 12, in <module>
    from marie.renderer import TextRenderer
  File "/home/rngem/Desktop/rms/marie-ai/marie/renderer/__init__.py", line 7, in <module>
    from .text_renderer import TextRenderer
  File "/home/rngem/Desktop/rms/marie-ai/marie/renderer/text_renderer.py", line 8, in <module>
    from marie.renderer import ResultRenderer
ImportError: cannot import name 'ResultRenderer' from partially initialized module 'marie.renderer' (most likely due to a circular import)

**Describe how you solve it**
this can happen when using code formatter (black) or isort

**Comment by Rithsek99** [comment]
skip `isort` to prevent this issue

**Comment by gregbugaj** [comment]
This have been already addressed via commit  [c8040afdbcbe0fd24fabe11d228a8291f2469310](https://github.com/marieai/marie-ai/commit/c8040afdbcbe0fd24fabe11d228a8291f2469310)

This not only requires `skip` but also the `depends`

Example:
```
from .toast_registry import Toast

from .native_handler import NativeToastHandler  # isort:skip depends on Toast
from .psql_handler import PsqlToastHandler  # isort:skip depends on Toast
from .rabbit_handler import RabbitMQToastHandler  # isort:skip depends on Toast
```
